src/NUnitEngine/nunit.engine/Internal/TcpChannelUtils.cs

```diff
@@ -123,7 +124,7 @@ public static TcpChannel GetTcpChannel(string name, int port, int limit, Current
                 }
                 catch (Exception ex)
                 {
-                    Log.Error("Failed to create/register channel", ex);
+                    Log.Error("Failed to create/register channel. {0}", ExceptionHelper.BuildMessageAndStackTrace(ex));
```

Comment:
I might have used concatenation instead of a format string, but this does stay consistent without changing everything.